js/freewheel.js

```diff
@@ -385,6 +386,7 @@ OO.Ads.manager(function(_, $) {
      */
     this.playAd = function(ad) {
       _resetAdState();
+      currentAd = ad;
```

Comment:
I did see the currentPlayingSlot variable. That get's assigned as ad.ad on line 407, which no longer has the isLinear field. Is there something similar to isLinear from ad.ad?